### PR TITLE
IDVA-911-Print a page for confirmation screen

### PR DESCRIPTION
**Pull request by atedros-ch**

### CODE REVIEW

**Review by ttingle-ch — APPROVED** [review]
LGTM

**Review by cbroadley-ch — APPROVED** [review]

File: src/views/partials/print_button.njk [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  text: "Print this page",
+  classes: "govuk-button--secondary",
+  href: "javascript: window.print()",
+  attributes: {
+      "id": "print-button",
+      "data-event-id": "print-button"
+  }
+}) }}
+
+<p class="govuk-body govuk-!-font-size-20 check-your-answers-javascript-disabled-text-style">To print this page, select print from your browser menu.</p>
```
Comment (swalke-ch) [inline src/views/partials/print_button.njk:29]: Is welsh translation needed for this?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**